delta/app/src/test/scala/ch/epfl/bluebrain/nexus/delta/routes/ProjectsRoutesSpec.scala

```diff
@@ -460,5 +476,31 @@ class ProjectsRoutesSpec
       }
     }
 
+    "fail to get the project statistics without resources/read permission" in {
+      Get("/v1/projects/org1/proj/statistics") ~> routes ~> check {
+        response.asJson shouldEqual jsonContentOf("errors/authorization-failed.json")
+        response.status shouldEqual StatusCodes.Forbidden
+      }
+    }
+
+    "fail to get the project statistics for an unknown project" in {
+      acls.append(Acl(AclAddress.Root, Anonymous -> Set(resources.read)), 9L).accepted
+      Get("/v1/projects/org1/unknown/statistics") ~> routes ~> check {
+        status shouldEqual StatusCodes.NotFound
+        response.asJson shouldEqual jsonContentOf("/projects/errors/project-not-found.json", "proj" -> "org1/unknown")
+      }
+    }
+
+    "get the project statistics" in {
+      Get("/v1/projects/org1/proj/statistics") ~> routes ~> check {
+        status shouldEqual StatusCodes.OK
+        response.asJson shouldEqual json"""{
+          "@context" : "https://bluebrain.github.io/nexus/contexts/statistics.json",
+          "instant" : "1970-01-01T00:00:00Z",
```

Comment:
I don't think "instant" is a clarifying enough name. Something like `lastProcessedEventDateTime`? That's what we currently have in the views statistics.